core/src/main/java/com/netease/arctic/trace/AmsTableTracer.java

```diff
@@ -175,8 +177,13 @@ public void commit() {
       update = true;
     }
     if (this.properties != null && Constants.INNER_TABLE_BASE.equals(innerTable)) {
-      commitMeta.setProperties(this.properties);
-      update = true;
+      try {
+        Map<String, String> catalogProperties = client.getCatalog(table.id().getCatalog()).getCatalogProperties();
+        commitMeta.setProperties(CatalogUtil.mergeCatalogPropertiesToTable(this.properties, catalogProperties));
```

Comment:
mergeCatalogPropertiesToTable may throw NPE  for catalogProperties may be null .

Reply:
fixed